twingate/internal/client/client.go

```diff
@@ -340,6 +343,19 @@ type ResponseWithPayload interface {
 	IsEmpty() bool
 }
 
+func (client *Client) queryWithTimeout(ctx context.Context, resp ResponseWithPayload, variables map[string]any, opr operation, attrs ...attr) error {
+	timeoutCtx, cancel := context.WithTimeout(ctx, defaultQueryTimeout)
```

Comment:
When context.WithTimeout is set outside the retry loop does that mean it has 10 seconds to complete including all retries?

If we set this inside the loop will each retry have 10 seconds to complete?

Reply:
thank you, fixed it